package.json

```diff
@@ -4,7 +4,11 @@
   "description": "hashmap + constructor function = a hashmap you don't have to check before calling get()",
   "main": "index.js",
   "scripts": {
-    "test": "tape test.js && jsmd readme.md"
+    "build": "rm -rf dist && npm test && npm run compile && cp package.json readme.md dist/",
```

Comment:
My publish scripts test before building, so I'd rather not run the test from the build script.

I'd rather not have the potential confusion of having the published directory be different from the repo directory – I'd rather just build to `./dist.js` or `./build.js` and put that filename into [`files`](https://docs.npmjs.com/files/package.json#files).